pool/static_pool/pool.go

```diff
@@ -201,8 +201,8 @@ begin:
 		rsp, err = w.Exec(context.Background(), p)
 	}
 
-	if sp.cfg.MaxJobs != 0 && w.State().NumExecs() >= sp.cfg.MaxJobs {
-		sp.log.Debug("max requests reached", zap.Int64("pid", w.Pid()))
+	if w.MaxExecsReached() {
+		sp.log.Debug("jobs execution limit reached, worker will be restarted", zap.Int64("pid", w.Pid()), zap.Uint64("execs", w.State().NumExecs()))
```

Comment:
make log more expressive

Reply:
No need to use the `jobs` term here. Since it might be confusing if user doesn't use the `jobs` plugin.

Reply:
Done